spec/functional/win32/registry_spec.rb

```diff
@@ -20,25 +20,6 @@
 require 'spec_helper'
 require 'chef/win32/registry'
 
-describe Chef::Resource::RegistryKey, :unix_only do
```

Comment:
This test is a duplicate of [this one](https://github.com/chef/chef/blob/master/spec/functional/resource/registry_spec.rb#L24-L41).